pkg/testcore/suites/helm-suites.go

```diff
@@ -47,6 +48,11 @@ func (ps *PostgresqlSuite) Run(ctx context.Context, storageClass string, clients
 		ps.VolumeSize = "32Gi"
 	}
 
+	if ps.Image == "" {
+		log.Info("Using default image")
```

Comment:
Can this say `log.Infof("Using default image: %s", "docker.io/bitnami/postgresql:11.8.0-debian-10-r72")`?